toptal/picasso
# feat(typography): add dark-grey color

### Description

The designs using light grey color actually use a bit darker color than originally implemented.

See example:
https://app.goabstract.com/projects/e7151a30-716b-11e9-aeb9-4db1a1401fcd/branches/master/commits/9ed546c127dd30240d964feb618eb89c066b9c48/files/2B00B4DA-0860-4748-A90F-95BB212C68D4/layers/4AA60E92-9F8C-44B4-9DA3-80F5618764CC?mode=build&nonce=812d80c1-f5f4-44a6-86ed-fe2265ace76c&selected=1257389566-EB9AF414-51A0-4A4A-B925-9BBE2666D77B

### How to test

- see new dark-grey variant
- see the grey variant of typography uses `#C4C6CA`

### Review
